pages/player/[playerName].tsx

```diff
@@ -48,6 +68,9 @@ export default function PlayerPage(props: PlayerPageProps) {
                     ' points)'
                 }
             />
+            <SortControl
+                onSelectedSortChange={setSelectedSort}
```

Comment:
You want to pass in the selectedSort value in here too so it's properly controlled, and then consume it as a `value` prop on the select.

Reply:
ill probably bug you to figure out what you mean for this one soon